test/Microsoft.Framework.PackageManager.FunctionalTests/KpmBundleTests.cs

```diff
@@ -1066,6 +1061,12 @@ public void BundleWithNoSourceOptionGeneratesLockFileOnClr(string flavor, string
                 var appPath = Path.Combine(tempDir, testApp);
                 TestUtils.CopyFolder(TestUtils.GetXreTestAppPath(testApp), appPath);
 
+                var lockFilePath = Path.Combine(appPath, "project.lock.json");
```

Comment:
Some other test cases might do `kpm restore` in the test app and generates a lockfile there. However, this test case is testing the scenario where a project doesn't have lockfile. Adding this to make it robust.